samples/net/prometheus/src/certificate.h

```diff
@@ -0,0 +1,50 @@
+/*
+ * Copyright (c) 2023 Nordic Semiconductor ASA
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#ifndef __CERTIFICATE_H__
+#define __CERTIFICATE_H__
```

Comment:
Not sure how much that matters, but it looked like you used the full path in the other #ifdef guards:
https://github.com/zephyrproject-rtos/zephyr/pull/72804/files#diff-3604770612d200b009418873ab10cdfc46a0c5ba37407de112ba896c0b6a4273R13

Reply:
Still applies, I think

Reply:
I can't remember what will i do. I just copied from :

https://github.com/zephyrproject-rtos/zephyr/blob/5e2203a6cdd0daf395bb3825d416843ecdc40629/samples/net/sockets/http_server/src/certificate.h#L7. Can you explain more?

Reply:
Yes, glad to! It looks like you had it done in one way [here](https://github.com/zephyrproject-rtos/zephyr/pull/72804/files#diff-3604770612d200b009418873ab10cdfc46a0c5ba37407de112ba896c0b6a4273R7-R8):
```
include/zephyr/prometheus/label.h   <- name of the file
ZEPHYR_INCLUDE_PROMETHEUS_LABEL_H_  <- name of the #ifdef guard
```

And in another way for this file `certificate.h`, but I forgot it was a sample, and there does not seems to be a very rigid way for samples:

```
~/zephyrproject/zephyr$ find samples/ -name '*.h' -exec grep -h '#ifndef ' {} + | sort -u | head -n 30
#ifndef APPLICATION_BATTERY_H_
#ifndef APP_SHARED_H
#ifndef APP_SYSCALL_H
#ifndef BYTE
#ifndef COMMON_H__
#ifndef CO_OD_H_
#ifndef DISPLAY_7SEG_HEADER
#ifndef ENC_H
#ifndef EXT_LOG_SYSTEM_ADAPTER_H
#ifndef EXT_LOG_SYSTEM_H
#ifndef FEEDBACK_H_
#ifndef MAIN_H
#ifndef PROD_CONSUMER_APP_A_H
#ifndef PROD_CONSUMER_APP_B_H
#ifndef RETAINED_H_
#ifndef SAMPLE_INSTANCE_H
#ifndef SAMPLE_MODULE_H
#ifndef STREAM_LC3_H
#ifndef STREAM_TX_H
#ifndef ST_BLE_SENSOR_BUTTON_SVC_H_
#ifndef ST_BLE_SENSOR_LED_SVC_H_
#ifndef TENSORFLOW_LITE_MICRO_EXAMPLES_HELLO_WORLD_CONSTANTS_H_
#ifndef TENSORFLOW_LITE_MICRO_EXAMPLES_HELLO_WORLD_MAIN_FUNCTIONS_H_
#ifndef THREAD_DEF_H
#ifndef ZEPHYR_FAKE_DRIVER_H
#ifndef ZEPHYR_SAMPLES_DRIVERS_LED_WS2812_F070RB_BINDINGS_H
#ifndef ZEPHYR_SAMPLES_DRIVERS_LED_WS2812_H_
#ifndef ZEPHYR_SAMPLES_SENSOR_SENSOR_SHELL_INCLUDE_TRIGGER_H
#ifndef ZEPHYR_SAMPLES_SUBSYS_USB_COMMON_SAMPLE_USBD_H
#ifndef _APP_GPIO_H
```

Nevermind then!